_rules/SC4-1-2-aria-state-or-property-has-valid-value.md

```diff
@@ -0,0 +1,276 @@
+---
+name: ARIA state or property has valid value
+test_type: atomic
+
+description: |
+   This rule checks that each ARIA state or property has a valid value
+   
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any [non-empty](#non-empty) [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element.
+
+### Expectation 1
+
+Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
+
+### Expectation 2
+
+For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) as the element that specifies the target attribute. 
+
+**Note:** This expectation only applies to [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState). For non-required properties, there is no requirement that the elements with the given ids actually exists.
+
+### Expectation 3
+
+For value type `URI` the value matches the [generic URI syntax](https://www.ietf.org/rfc/rfc3986.txt).
+
+**Note:** 
+For value type `URI`, this rule does not require that the destination URI exists. 
```

Comment:
These expectations are not valid according to ACT as they are not distinct; expectation 2 and 3 are contained within expectation 1.

Reply:
@nitedog, see Kasper's comment for reference as to why I change this back to one big exxpectation.